ScreenControl_Unity/Assets/ScreenControl/Service/Scenes/ScreenControlService.unity

```diff
@@ -191,7 +191,7 @@ PrefabInstance:
     - target: {fileID: 2213489624516412116, guid: 157025b17471e7c41a7d20e7f28273b5,
```

Comment:
Can some of these scene changes be prefab changes instead?

Reply:
Ok so just looking over the changes:

* The Auto-config screen gets a reference to the cursor on the Client prefab. Because this is across prefabs, it has to be scene-based
* ScreenControl Service prefab gets a "disableOnFocusLost" script, which needs a reference to the Screen Control Client to disable/enable it on focus (because it can't go on the Client prefab because then it'd be disabled when focus is lost... and never come back.) I think I'd rather leave this off the prefab because it's not a general use-case and is specific to the Service.

Reply:
Okay that sounds fine then 👍